nym-vpn-app/src-tauri/src/window.rs

```diff
@@ -82,7 +82,7 @@ impl AppWindow {
         .center()
         .focused(true)
         .inner_size(328.0, 710.0)
-        .min_inner_size(160.0, 200.0)
+        .min_inner_size(328.0, 200.0)
```

Comment:
This is a bad idea:
The user's monitor DPI is not predictable and user can set whatever in-app UI scaling.
In some situations (Linux and high or very low DPI monitors) users would want to resize down the window width as needed.
And the UI is responsive enough to be squished.